djangogirls/settings.py

```diff
@@ -94,7 +94,7 @@ def gettext(s):
     default=f'postgres://{POSTGRES_USER}:{POSTGRES_PASSWORD}@{POSTGRES_HOST}:{POSTGRES_PORT}/{POSTGRES_DB}'
 )
 
-DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
```

Comment:
Ok I think this is the only area of concern for this change.

With the upgrade to 3.2 we adopted the future default, `BigAutoField`, and migrated all models. So to go back to `AutoField` seems like a backward step, because we'll have to go back to `BigAutoField` in future.

Reply:
Ok, let me remove it then. Thanks for that.

Reply:
Ok, great! I've added comments to each of the migration files that you can delete after changing this. Hopefully that's helpful.

The organize app also has two migrations after the one to delete so you could rename those to 8 & 9 or delete them & recreate with `makemigrations`